### DISCUSSION TITLE
One more unhandeled exception

### DISCUSSION THREAD

**Discussion by spamkutu** [Chia-Network/chia-blockchain#8322] [Support]
my hardvester cannot see any pilot and its reclame add pliot with the flower picture just like start i restarted cmd much no effect
i were also restarted gui with no change since than i cant restart becouse i had to wait my five pilot finish first its almost %80 arround but not going much so so slow hope its all done until the morning and i could restart my windows too and hope after restart it will be all good and my hardvester reseen my pilots too i am hopeless atm and hope my pilots could finish to i have half chance very weird :(

here below the err reply caused exception bug.
please find a clue why that thats and fix as soon thanks.
-------------------------
C:\Users\INT>chia start farmer -r
chia_harvester: started
chia_farmer: Error, timeout.
stop failed
chia_farmer: Task exception was never retrieved
future: <Task finished name='Task-5' coro=<DaemonProxy.start.<locals>.listener() done, defined at chia\daemon\client.py:29> exception=ConnectionClosedError('code = 1006 (connection closed abnormally [internal]), no reason')>
Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "chia\daemon\client.py", line 32, in listener
  File "websockets\protocol.py", line 509, in recv
  File "websockets\protocol.py", line 812, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
Error, timeout.
chia_farmer failed to start. Error: no response
Task exception was never retrieved
future: <Task finished name='Task-22' coro=<WebSocketCommonProtocol.send() done, defined at websockets\protocol.py:521> exception=ConnectionClosedError('code = 1006 (connection closed abnormally [internal]), no reason')>
Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
asyncio.exceptions.CancelledError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "websockets\protocol.py", line 555, in send
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
Error, timeout.
Traceback (most recent call last):
  File "D:\a\chia-blockchain\chia-blockchain\venv\Lib\site-packages\chia\cmds\chia.py", line 132, in <module>
  File "D:\a\chia-blockchain\chia-blockchain\venv\Lib\site-packages\chia\cmds\chia.py", line 128, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\start.py", line 14, in start_cmd
  File "asyncio\base_events.py", line 642, in run_until_complete
  File "chia\cmds\start_funcs.py", line 60, in async_start
  File "chia\daemon\client.py", line 85, in is_running
TypeError: 'NoneType' object is not subscriptable
[102560] Failed to execute script 'chia' due to unhandled exception!

**Comment by loppefaaret** [comment]
let us know how it goes after a reboot ?

**Reply by spamkutu** [reply]
reboot not efect did 3-5 times but after deleted config.yaml that see my pilots finaly but this time i lost my pools.

**Reply by spamkutu** [reply]
my pilots notice me working produce but im not sure how that possible without see any pool, thats all were the pool pilots is that realy working for their pools now or working for like self pool or non working but showing produce fake info.

Anyway i need and i have to recall my pools contrats back

reboots, and delete config.yaml not solved this i tryed alot.

**Comment by spamkutu** [comment]
After deleted config.yaml  and restart my system finaly my pilots back after minutes its all seem good and working progress but without the pools. This time i lost my pools restarted gui, restarted windows waited so long over 8 - 10 hours no corrected. Now i cant produce new pilot on this hardvester becouse i could not see any pools inside it ( at least with gui) and im not sure how my pilots working now  is that realy working progess without the see any pools?? is that working like self pooling or joined pools works well or that showing work progress not real and non of my pilots working now??

how could i call my pools back? Please help urgent help so many thanks whoever give me a clue or solve this..

Regards,
SK.

**Reply by spamkutu** [reply]
1 days pass still show me join a pool state to me. i am in some pools allready but not show in the gui my plot works like produce but i not belive becouse of no any pool on the screen just join a pool state.

**Reply by loppefaaret** [reply]
stop removing `config.yaml`
close chia and remove the wallet database of the wallet that owns the plotnft's that are missing, and opne chia again and wait for it to sync wallet - then they will come back

**Comment by spamkutu** [comment]
i fixed some but not all yet how i did i copied some lines from the second harvester to my first farmer than its saw without delete my wallet and wait to sync so long but i guess there is problem since without sync its permis to join pools  (i guess it musn't do that untill full sync) becouse whenever join the pools you cant see nothing (and off course with a pay something)  becouse of that you could apply so many times to join pool becouse of unsync state i saw my config entry thats totaly 34 pools requested but only mostly see 18 of them sometimes its show only 8to12 but whenever try to close and open chia app its show me the most 18 pool so becouse i paid and not want to lose them i piloted atleast 3-5 times to my all nfts (until my first 18 nfts ; thats totaly show on 34 on config yaml but i think after 18 nft i got err and joined pools more to more some ) now result: nft admin must be renewed or atleast until the full sync it must not be permist to join a pool (until the sync its always asked me and i made jump pools but no see so rejump jump jump its happen alot of nft)
as you know if your nfts inaccecile chia not let you change your pilots to other nfts so you could lose all your work. So reliability of the nft pool side of chia must be rexamine carefuly. if you say you could get all your nfts back after the delete wallet db it could good news and i could try but that db also to long now i dont think if i delete that db i have to wait 1 weeks or more i guess so i not touch that yet.

i consider you all you must back up .chia directory atleast config files often to not much trouble like me happy chiassss...

**Comment by BrandtH22** [comment]
@spamkutu , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
